docker-compose-full-cm.yml

```diff
@@ -0,0 +1,444 @@
+      CSM_TARGET_SMTP_SERVER_ADDRESS: ${CSM_TARGET_SMTP_SERVER_ADDRESS}
+      CSM_TARGET_SMTP_SERVER_PORT: ${CSM_TARGET_SMTP_SERVER_PORT}
+
+      TIMEZONE: ${TIMEZONE}
+      LOGGING_LEVEL: ${API_LOGGING_LEVEL}
+    logging:
+      options:
+        max-size: '10m'
+        max-file: '5'
+
+  cvmanager_webapp:
+    build:
+      context: webapp
+      dockerfile: Dockerfile
+      args:
+        API_URI: http://${WEBAPP_DOMAIN}:8081
+        MAPBOX_TOKEN: ${MAPBOX_TOKEN}
+        KEYCLOAK_HOST_URL: http://${KEYCLOAK_DOMAIN}:8084/
+        COUNT_MESSAGE_TYPES: ${COUNTS_MSG_TYPES}
+        VIEWER_MESSAGE_TYPES: ${VIEWER_MSG_TYPES}
+        DOT_NAME: ${DOT_NAME}
+        MAPBOX_INIT_LATITUDE: ${MAPBOX_INIT_LATITUDE}
+        MAPBOX_INIT_LONGITUDE: ${MAPBOX_INIT_LONGITUDE}
+        MAPBOX_INIT_ZOOM: ${MAPBOX_INIT_ZOOM}
+        CVIZ_API_SERVER_URL: ${CVIZ_API_SERVER_URL}
+        CVIZ_API_WS_URL: ${CVIZ_API_WS_URL}
+    image: jpo_cvmanager_webapp:latest
+    restart: always
+    depends_on:
+      cvmanager_keycloak:
+        condition: service_healthy
+    extra_hosts:
+      ${WEBAPP_DOMAIN}: ${WEBAPP_HOST_IP}
+      ${KEYCLOAK_DOMAIN}: ${KC_HOST_IP}
+    ports:
+      - '80:80'
+    logging:
+      options:
+        max-size: '10m'
+
+  cvmanager_postgres:
+    image: postgis/postgis:15-master
+    restart: always
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_USER: ${PG_DB_USER}
+      POSTGRES_PASSWORD: ${PG_DB_PASS}
+    volumes:
+      - pgdb:/var/lib/postgresql/data
+      - ./resources/sql_scripts:/docker-entrypoint-initdb.d
+    logging:
+      options:
+        max-size: '10m'
+
+  cvmanager_keycloak:
+    build:
+      context: ./resources/keycloak
+      dockerfile: Dockerfile
+      args:
+        KEYCLOAK_LOGIN_THEME_NAME: ${KEYCLOAK_LOGIN_THEME_NAME}.jar
+    image: jpo_cvmanager_keycloak:latest
+    restart: always
+    depends_on:
+      - cvmanager_postgres
+    extra_hosts:
+      ${WEBAPP_DOMAIN}: ${WEBAPP_HOST_IP}
+      ${KEYCLOAK_DOMAIN}: ${KC_HOST_IP}
+    ports:
+      - '8084:8080'
+    environment:
+      KEYCLOAK_ADMIN: ${KEYCLOAK_ADMIN}
+      KEYCLOAK_ADMIN_PASSWORD: ${KEYCLOAK_ADMIN_PASSWORD}
+      WEBAPP_ORIGIN: http://${WEBAPP_DOMAIN}
+      WEBAPP_CM_ORIGIN: http://${WEBAPP_CM_DOMAIN}
+      KC_HEALTH_ENABLED: true
+      KC_DB: postgres
+      KC_DB_URL: jdbc:postgresql://${PG_DB_HOST}/postgres?currentSchema=keycloak
+      KC_DB_USERNAME: ${PG_DB_USER}
+      KC_DB_PASSWORD: ${PG_DB_PASS}
+      KC_HOSTNAME: ${KEYCLOAK_DOMAIN}
+      KEYCLOAK_API_CLIENT_SECRET_KEY: ${KEYCLOAK_API_CLIENT_SECRET_KEY}
+      KEYCLOAK_CM_API_CLIENT_SECRET_KEY: ${KEYCLOAK_CM_API_CLIENT_SECRET_KEY}
+      GOOGLE_CLIENT_ID: ${GOOGLE_CLIENT_ID}
+      GOOGLE_CLIENT_SECRET: ${GOOGLE_CLIENT_SECRET}
+    command:
+      - start-dev
+      - --log-level=${KC_LOGGING_LEVEL}
+      - --import-realm
+      - --spi-theme-welcome-theme=custom-welcome
+    logging:
+      options:
+        max-size: '10m'
+
+  zookeeper:
```

Comment:
Is zookeeper really necessary still? Looks like you removed it from the main docker-compose so this may have been missed.

Reply:
re-synchronized this with the zookeeper-less kafka updates